src/geoip2/records.py

```
@@ -262,11 +262,59 @@ def __init__(self, *, queries_remaining: int | None = None, **_: Any) -> None:
+    """The name of the provider associated with the network in this
+    anonymizer feed. This attribute is only available from the Insights end
+    point.
+    """
+
+    def __init__(
+        self,
+        *,
+        confidence: int | None = None,
+        network_last_seen: str | None = None,
+        provider_name: str | None = None,
+        **_: Any,
```

Ruff (0.15.21)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `**_`

(ANN401)